tabs/configuration.js

```diff
@@ -140,13 +148,39 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
     function load_name() {
         var next_callback = load_rx_config;
+
+        if (self.SHOW_OLD_BATTERY_CONFIG) {
+            next_callback = load_battery;
+        }
+
         if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
             MSP.send_message(MSPCodes.MSP_NAME, false, false, next_callback);
         } else {
             next_callback();
         }
     }
 
+    // -----------------------------------------------
```

Comment:
This is confusing. If only what you have marked as 'remove' is removed, line 153 will cause an exception.